### ISSUE TITLE
Ripper Request: Rip from Date Posted

### ISSUE THREAD

**Issue by stacalkas** [RipMeApp/ripme#1494]
Sorry if isn't labeled or in the right place.

An option to rip based on a range of posts based on their dates. For Reddit, if at all possible. Thanks.

**Comment by madhatr** [comment]
This could be an interesting feature, except it would be sure to have done a complete rip initially and then proceed to only rip from the last time ripping was done.

**Comment by Malkierian** [comment]
No, I don't think having a previously complete rip would or should be necessary for this. On Instagram, for instance, users change their usernames quite often (that's something else that would be nice on here, store based on IG user ID instead of username. but that's a separate issue), and so, when that happens, adding their new username to the list with a manual date would be very advantageous.
